tools/pontoon/search_filters.md

```diff
@@ -14,6 +14,8 @@ Strings in Pontoon can be filtered by their status. A string can be in one of th
 * *Rejected*: string has been reviewed and rejected by a translator.
 
 There are some extra criteria that can be used to filter strings:
+* *Warnings*: string contains issues classified as [warnings](translate.md#warnings).
+* *Errors*: string contains [critical issues](translate.md#errors).
```

Comment:
These two bullet points should go under **String status** section.

The **Untranslated** should also be updated to include Errors and Warnings. Also in the **Example workflow** section.